packages/widget-display-mode/src/DisplayModeWidget.vue

```diff
@@ -52,6 +52,12 @@
         } else {
           this.setNormalLayout();
         }
+        const query = {
+          ...this.$route.query,
+          mode: mode.mode,
+        };
+        delete query['rs'];
```

Comment:
I think if / when we get to a point where we need to track more than one bit of "display mode state", we might want to "register / reserve" querystring params; I think this is fine for now.

Reply:
my thinking is that `rs` will hold all "reader state" and we can do some type of base64 encoding of a key/value pair delimited string

something like:

```
> var settings = {mode: 'image-folio', zoomLevel: 3, textSize: 'XL'}
> btoa(JSON.stringify(settings))
"eyJtb2RlIjoiaW1hZ2UtZm9saW8iLCJ6b29tTGV2ZWwiOjMsInRleHRTaXplIjoiWEwifQ=="
> JSON.parse(atob('eyJtb2RlIjoiaW1hZ2UtZm9saW8iLCJ6b29tTGV2ZWwiOjMsInRleHRTaXplIjoiWEwifQ'))
Object { mode: "image-folio", zoomLevel: 3, textSize: "XL" }
```

putting the results, in this case `eyJtb2RlIjoiaW1hZ2UtZm9saW8iLCJ6b29tTGV2ZWwiOjMsInRleHRTaXplIjoiWEwifQ==` in the `rs` querystring parameter.